examples/frontend/pages/devs/index.tsx

```diff
@@ -23,7 +24,7 @@ function DefaultColumnFilter({
 
     return (
         <input
-            value={filterValue || ''}
+            value={filterValue || ``}
```

Comment:
https://github.com/curaOS/source/blob/3cf4c52feccd322eab164e40a96f3b52fe7f13cd/examples/frontend/.eslintrc#L27

Reply:
I think it would be great if it's standarlized because of it's advantages over the normal quotes.
But I guess it has downsides to performance because it would make parsing them harder.

I can change the `eslint` and undo them if that's better

Reply:
Seems like I've put it there months ago. Thanks!